app/components/widgets/settings/radio_setting.js

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {Text, TouchableOpacity, View} from 'react-native';
+import CheckMark from 'app/components/checkmark';
+import {paddingHorizontal as padding} from 'app/components/safe_area_view/iphone_x_spacing';
+import FormattedText from 'app/components/formatted_text';
+import {makeStyleSheetFromTheme, changeOpacity} from 'app/utils/theme';
+
+export default class RadioSetting extends PureComponent {
+    static propTypes = {
+        id: PropTypes.string.isRequired,
+        label: PropTypes.node.isRequired,
+        values: PropTypes.array.isRequired,
+        value: PropTypes.string.isRequired,
+        onChange: PropTypes.func.isRequired,
+        theme: PropTypes.object.isRequired,
+        isLandscape: PropTypes.bool.isRequired,
+        helpText: PropTypes.node,
+        optional: PropTypes.bool,
+        showRequiredAsterisk: PropTypes.bool,
+    };
+
+    static defaultProps = {
+        optional: false,
+        showRequiredAsterisk: false,
+        isLandscape: false,
+    };
+
+    handleChange = (item) => {
+        this.props.onChange(this.props.id, item);
```

Comment:
shoulnd't we change where value points to? so the mark moves to the pressed value